src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -397,7 +397,7 @@ const matchingLineNumbers = ref<number[]>([])
 const currentLineIndex = ref<null | number>(null)
 
 const totalLines = computed((): number[] => Array.from({ length: props.code?.split('\n').length }, (_, index) => index + 1))
-const maxLineNumberWidth = computed((): string => totalLines.value[totalLines.value.length - 1]?.toString().length + 'ch')
+const maxLineNumberWidth = computed((): string => totalLines.value[totalLines.value?.length - 1]?.toString().length + 'ch')
```

Comment:
👋 Drive-by comment: Is this a bug fix, and if so should it go in a different PR? If not should it be here?

Reply:
Will there be a separate changelog entry for folks doing dependabot PRs and reading changelogs to see what has changed?

Reply:
No but a question mark check is hardly a change anyone would be curious about I imagine

Reply:
Oh I thought it was a bug fix?

Reply:
Well, DD error fix. Does not affect users.

Reply:
oki doki, feel free to resolve!